core-blocks/gallery/style.scss

```diff
@@ -38,6 +38,10 @@
 			width: 100%;
 			max-height: 100%;
 			overflow: auto;
+
+			img {
+				display: inline;
```

Comment:
Any way we could somehow have a "front-end" style sheet in the inline block registrations that gets included/imported here? Probably thinking too much of SCSS as JS haha. Would be cool if we could just do

```scss
figcaption {
  ...

  &inlineBlockStyles
}
```

or something. Ignore me I know nothing about SCSS 🙈

Maybe @jasmussen knows some magic here to separate the styles.

Reply:
Ooooooh, something like this maybe? https://sass-lang.com/guide#topic-6

Reply:
```scss
figcaption {
  ...

  @include inline-block-styles();
}
```

Reply:
Not entirely sure about the context for this code, but if the goal is to reuse bits of CSS then yes, I think a mixin can be the right way to do it. We already do this for button hover/focus styles: https://github.com/WordPress/gutenberg/blob/master/edit-post/assets/stylesheets/_mixins.scss#L129

Reply:
Yes, these would be the styles from all inline blocks together. Not sure about the best solution here, maybe the current one is good for a first version.

Reply:
Interesting thoughts here. I like the idea of using a mixin, but not sure if it's possible to combine a group of stylesheets into a single mixin. Perhaps examining how standard blocks add styles would be instructive here. It doesn't look like they use mixins, but instead each have their own stylesheets ([Image block for example](https://github.com/WordPress/gutenberg/blob/master/core-blocks/image/editor.scss)). Blocks are also given a [unique className](https://github.com/WordPress/gutenberg/blob/master/docs/block-api.md#supports-optional) -- maybe inline blocks could have a className property as well.

Reply:
I think this is fine to figure out later.